src/backend/app/db/models.py

```diff
@@ -14,3 +14,15 @@ class AnnotationHexagon(Base):
     annotation_id = Column(Integer, ForeignKey("annotation.annotation_id"))
     hex_index = Column(String)
     hex_type = Column(String)
+
+class Prediction(Base):
+    __tablename__="prediction"
+    prediction_id = Column(Integer, primary_key=True, index=True)
+    taxa_id = Column(Integer)
+
```

Comment:
I think we can ignore this for now. The database proposal has this linked via a foreign key to the `model` table with the `created_at` stored for the model. Since we haven't implemented that table yet, I think it's okay to leave out creation times for now.

Reply:
The created_at column is not in the MVB database design (see https://docs.google.com/document/d/1yryxqj19b_SocJbYjW3NjGAeexVmYe1-DRdXUD8ed5Q/edit), but it's nice to have, will add i.

Reply:
Added!